packages/terra-form-select/src/combobox/Menu.jsx

```diff
@@ -299,30 +297,23 @@ class Menu extends React.Component {
     return this.state.active === this.props.value;
   }
 
-  updateNoResultsScreenReader() {
+  updateNoResultsScreenReader(freeTextValue) {
     if (this.liveRegionTimeOut) {
       clearTimeout(this.liveRegionTimeOut);
     }
 
     this.liveRegionTimeOut = setTimeout(() => {
-      const { hasNoResults } = this.state;
-
       const {
         intl,
         visuallyHiddenComponent,
         searchValue,
       } = this.props;
-
       // Race condition can occur between calling timeout and unmounting this component.
       if (!visuallyHiddenComponent || !visuallyHiddenComponent.current) {
         return;
       }
-
-      if (hasNoResults) {
-        visuallyHiddenComponent.current.innerText = intl.formatMessage({ id: 'Terra.form.select.noResults' }, { text: searchValue });
-      } else {
-        visuallyHiddenComponent.current.innerText = '';
-      }
+      const noMatchingResultText = intl.formatMessage({ id: 'Terra.form.select.noResults' }, { text: searchValue });
+      visuallyHiddenComponent.current.innerText = `${noMatchingResultText}, ${freeTextValue}`;
```

Comment:
Can you attach JAWS speech history as well similar to VO screenshots as a evidence

Reply:
Done.

Reply:
Done. Thanks